### DISCUSSION TITLE
feed back on achievement badges

### DISCUSSION THREAD

**Discussion by SteveALee** [community/community#21073] [Other Feature Feedback, Questions, & Ideas]
Please turn these off by default. Gamification has no place here. Useless twaddle.

**Comment by jgmac1106** [comment]
i agree on off by default. Always default to privacy. 

I disagree on utility. If the achievements had useful metadata that complied with current industry recommendations the information could be ingested to track role based training requirements of developers, aid in portfolio reviews, and allow users to control their learning data outside of employers.

Granted the achievements (little disappointing) are just images for now, but it could be easily extendable to allow parsing, ignestiong, and  recording in an immutable ledger.

**Reply by SteveALee** [reply]
That's an interesting idea but a big "if" to get the badges representing meaningful development metrics rather than feel good.

**Reply by ghost** [reply]
Gamifying something just to sell more metadata to corporations who parasitize human privacy is inherently exploitation.  The purpose is to manifest addictive behaviour with immaterial rewards. Which is a manipulative function of industrial psychology, though common practice in today's ecosystems of consumer spyware and vacuous social media.

**Comment by MrSarno** [comment]
I'm not sure whether you're aware, but there is a setting to disable them [here](https://github.com/settings/profile#profile-settings-heading).

I don't feel strongly about your suggestion one way or the other. I think people are more likely to check for settings to disable features they dislike than they are to search for features they might hypothetially like and wish to enable. In addition, some people may not think to look for settings, and so there would likely be a significant number of people reporting the lack of achievements as a bug, and / or creating discussions to ask why they don't appear to be working. At least the setting's there for those who want it.

**Reply by MlgmXyysd** [reply]
I agree with this point

**Reply by mark-i-m** [reply]
Thanks for this! I would never have found that setting on my own.

**Comment by SteveALee** [comment]
OK, personally I rarely wanyt to spend time time hunting for options to
turn off new features I don't want that got turned on.

I know discovery can be an issue, but better to review the list of new
features when I have some bandwidth.

Thanks for the pointer

**Comment by johnno1962** [comment]
Can we cut it out with these new "achievement" badges already? I am now being shamed for "closing a PR within 5 minutes of opening it" and "merging a PR to my own repo without a review". This is all just puerile. Make it opt-in and grow up.

**Comment by return42** [comment]
Remove this "feel good" crap.

**Comment by suchov** [comment]
I love them! ❤️

**Reply by return42** [reply]
For what reason? It is not clear to me what value this information has.

**Reply by johnno1962** [reply]
Was it really necessary to put a "YOLO" badge against my account because I didn't wait out a reviewer? What it is supposed to say about me? What purpose does saying that serve? According to google: "YOLO" is an acronym for "you only live once". Along the same lines as the Latin carpe diem, it is a call to live life to its fullest extent, even embracing behavior which carries inherent risk. It became a popular internet slang term in 2012.

**Reply by suchov** [reply]
Repo with many stars, many PR's merged - can be a stimulus to newbies ...

**Reply by return42** [reply]
When "many" means a value of 10 or 15? It feels like kindergarten, where everyone gets a pink unicorn badge. Everyone is happy and the parents of each child are proud.

**Reply by suchov** [reply]
So your suggestion is to higher the bar to get the badge?

**Reply by return42** [reply]
IMO all badges lack sustainable information. 

The one you mentioned *".. can be a stimulus to newbies .."* might be reasonable .. but what ever limit gh sets .. the users will find a way to increase their ratings / like mentioned [here]( https://github.com/orgs/github-community/discussions/21073#discussioncomment-3214608) or [here](https://github.com/orgs/github-community/discussions/21073#discussioncomment-3218604) .. I already have seen bots faking the activity on gh ..

**Comment by piazzai** [comment]
I completely agree. Unfortunately the chances that they roll back a feature that was already deployed in beta are very low. This should have been stopped at an earlier stage of the development process. Though the amount of users out there who are currently trading stars, answering each other's bogus discussions, coordinating to get the pair achievement, etc., should be a strong indication for product managers that they really didn't think this one through.

**Comment by ilechuks73** [comment]
I love it, now there's colors, lots of colors on my github profile. the only issue that may arise from this is having a cluttered UI if more features like this are getting rolled out. The senior and veteran software engineer might not need this but for the entry level developers like me whose portfolio is their github profile, it helps spice things up.

**Comment by ghost** [comment]
I hope also they don't add more, because it will rush a lot of people doing trivial things or doing nothing to get one.
Proof of this is a large number of empty discussions to get an achievement
But I do not deny that these achievements have pushed me forward, to make a lot of contribution

**Comment by jfullstackdev** [comment]
as always, there are opposing views here but the most important thing is it can be hidden at the Settings, a user can even set his / her profile to private, so no gamification at all

**Comment by sfertman** [comment]
Agree, this is such a useless waste of time. git is for code sharing and colab. Stop trying to make github into facebook/twitter. It's a distraction and will never work.

**Comment by johnno1962** [comment]
This is the sort of b.s. badges incentivise: https://github.com/johnno1962/injectionforxcode/pull/303 and repo owners have to put up with. This frivolity doesn't come at zero cost.

**Comment by ghost** [comment]
This kind of 'gamification' is purposefully designed to exploit addictive behaviour with immaterial social media internet points.

There's no denying the psychological impact these 'banana stickers' are intended to have.  They are intended to reward and/or punish people within the scope of social media, which git is not supposed to be.  By getting people addicted to chasing meaningless  -- and even harmful -- activity, you're getting people addicted to the platform itself.

That is, of course, the real intent:  Capitalist monopolization through psychological exploitation and (esp in the case of Microsoft) owning every damn thing, software, idea, concept, material work, organization, means of communication, collaborative space, etc. there is.

The secondary purpose here is creating and visually quantifying user metadata, which is being gathered on users (inherently anti-privacy) and sold as a product.  

The concept of open-source was never to monopolize, spy on, or manipulate people.  How did our tools for collectivism, cooperation and public good, merely become an opportunity for billionaire plutocrats to turn human privacy into just another thing to sell?  Surveillance capitalism, that's how.
